app/src/main/java/com/github/se/cyrcle/ui/theme/molecules/Filter.kt

```diff
@@ -371,6 +381,24 @@ fun SearchBarListScreen(
       animateDpAsState(targetValue = if (isTextFieldVisible.value) 0.dp else (-200).dp)
 
   Box {
+    // Callback for when a suggestion is clicked. If address is null, the user's location is set.
```

Comment:
Can address be null? I understand and appreciate that you handle edge cases here but I'm not sure what that would signify (and why the user's location is an appropiate fallback in such a case)...

Reply:
This was to indicate that calling it with an address sets the address but calling it with null (explicitely) sets the user location. See the usages on lines 434 and 479

Reply:
Do you think I should update the documentation to make it clearer? Or is just seeing the calls enough?

Reply:
Updated documentation is always good :). But honestly what you've written is clear enough

Reply:
Updated!

Reply:
Thanks!